src/app/visual/error/error-visual.component.html

```diff
@@ -57,6 +57,21 @@
   </sky-error>
 </div>
 
+<div class="error-screenshot" id="screenshot-error-custom-default-image">
+  <sky-error>
+    <sky-error-image
+      imageType="broken"
+    >
+    </sky-error-image>
+    <sky-error-title>
+      custom title
+    </sky-error-title>
+    <sky-error-description>
+      custom description
+    </sky-error-description>
```

Comment:
Let's throw the `<sky-error-action>` in here (like the other examples), just to be sure.